frontend/src/modules/TimeSeriesParameterDistribution/queryHooks.tsx

```diff
@@ -19,125 +19,62 @@ export function useVectorsQuery(
     caseUuid: string | undefined,
-    ensembleName: string | undefined,
-
+    ensembleName: string | undefined
 ): UseQueryResult<EnsembleParameterDescription_api[]> {
     return useQuery({
-        queryKey: [
-            "getParameterNamesAndDescription",
-            caseUuid,
-            ensembleName,
-        ],
-        queryFn: () =>
-            apiService.parameters.getParameterNamesAndDescription(
-                caseUuid ?? "",
-                ensembleName ?? "",
-            ),
+        queryKey: ["getParameterNamesAndDescription", caseUuid, ensembleName],
+        queryFn: () => apiService.parameters.getParameterNamesAndDescription(caseUuid ?? "", ensembleName ?? ""),
         staleTime: STALE_TIME,
         cacheTime: CACHE_TIME,
         enabled: caseUuid && ensembleName ? true : false,
     });
 }
 
-
 export function useParameterQuery(
     caseUuid: string | undefined,
     ensembleName: string | undefined,
-    parameterName: string | undefined,
-
-
+    parameterName: string | undefined
 ): UseQueryResult<EnsembleParameter_api> {
     return useQuery({
-        queryKey: [
-            "getParameter",
-            caseUuid,
-            ensembleName,
-            parameterName,
-
-
-        ],
-        queryFn: () =>
-            apiService.parameters.getParameter(
-                caseUuid ?? "",
-                ensembleName ?? "",
-                parameterName ?? "",
-            ),
+        queryKey: ["getParameter", caseUuid, ensembleName, parameterName],
+        queryFn: () => apiService.parameters.getParameter(caseUuid ?? "", ensembleName ?? "", parameterName ?? ""),
         staleTime: STALE_TIME,
         cacheTime: CACHE_TIME,
         enabled: caseUuid && ensembleName && parameterName ? true : false,
     });
 }
-export function useVectorAtTimestepQuery(
+export function useVectorAtTimestampQuery(
     caseUuid: string | undefined,
     ensembleName: string | undefined,
     vectorName: string | undefined,
-    timeStep: string | null
-
-
+    timestampUtcMs: number | null
 ): UseQueryResult<EnsembleScalarResponse_api> {
     return useQuery({
-        queryKey: [
-            "getRealizationVectorAtTimestep",
-            caseUuid,
-            ensembleName,
-            vectorName,
-            timeStep
-
-
-        ],
+        queryKey: ["getRealizationVectorAtTimestep", caseUuid, ensembleName, vectorName, timestampUtcMs],
         queryFn: () =>
-            apiService.timeseries.getRealizationVectorAtTimestep(
+            apiService.timeseries.getRealizationVectorAtTimestamp(
                 caseUuid ?? "",
                 ensembleName ?? "",
                 vectorName ?? "",
-                timeStep ?? ""
+                timestampUtcMs ?? 0
             ),
         staleTime: STALE_TIME,
         cacheTime: CACHE_TIME,
-        enabled: caseUuid && ensembleName && vectorName && timeStep ? true : false,
+        enabled: !!(caseUuid && ensembleName && vectorName && timestampUtcMs != undefined),
```

Comment:
`timestampUtcMs` is of type `number | null`, not `undefined`. Why `!!`? Is that a special operator I never heard of or double negation?

Reply:
I'll change it to `timestampUtcMs != null`
`!!` is double negation which ensures we're left with a boolean.